layouts/plugins/single.html

```diff
@@ -17,29 +17,67 @@
           </div>
         </div>
       </div>
+      <div class="col-lg-6">
+        <div class="plugins-small__field-wrapper input-group mb-3">
+          <div class="plugins-small__chips">
+          </div>
+          <div class="plugins-small__select" onclick="onSelect()">
+            {{ .Params.filter_title }}
+            <ul class="plugins-small__select-wrapper">
+              {{ range .Params.select_types }}
+              <li>
+                <label for="{{ . }}">
+                  <input type="checkbox" id="{{ . }}" class="plugins-small__checkbox" name="type" value="{{ . }}"/>
+                  <div class="checkmark"></div>
+                  {{ humanize . }}
+                </label>
+              </li>
+              {{ end }}
+            </ul>
+          </div>
+        </div>
+      </div>
     </div>
   </div>
   <div class="container">
     <div class="row">
       <div class="col-12">
-        <h2 class="plugins__title"></h2>
+        <div class="plugins__quantity-wrapper" style="display: none">
+          <span class="plugins__quantity"></span> Plugin displayed
```

Comment:
somehow the space doesn't show up in the rendering:
![image](https://user-images.githubusercontent.com/3895099/55760442-f3eec980-5a10-11e9-9b19-eb7944af19cf.png)

Also, lower case ``p`` and make plural: ``plugins``

Reply:
done

Reply:
done.